src/dlls/mscorpe/ceefilegenwriter.cpp

```diff
@@ -420,13 +427,13 @@ HRESULT CeeFileGenWriter::generateImage(void **ppImage)
 
     if (! outputFileName && ppImage == NULL) {
         if (m_comImageFlags & COMIMAGE_FLAGS_IL_LIBRARY)
-            outputFileName = L"output.ill";
+            outputFileName = outputFileNameIll;
```

Comment:
Could you please fix the write method to accept const string instead as the filename? There is no reason for it to modify it.